docs/install.rst

```diff
@@ -48,7 +48,7 @@ GWpy has the following strict requirements:
    ==================  ===========================
    |astropy|_          ``>=4.0``
    |dqsegdb2|_
-   |gwdatafind|_
+   |gwdatafind|_       ``>=1.1.0``
```

Comment:
Should this be 1.1.1? https://chat.ligo.org/ligo/pl/6o16hqq333rh9bbniktyppr5dh

Reply:
No, because the bug that prompted 1.1.1 is only in the `gwdatafind.http` interface, which is exactly what is being replaced here.